picard/tagger.py

```diff
@@ -1437,6 +1441,10 @@ def process_picard_args():
     parser.add_argument("-display", nargs=1, help=argparse.SUPPRESS)
 
     # Picard specific arguments
+    parser.add_argument("-a", "--audit", action='store',
```

Comment:
Should we check if `sys.addaudithook` is available and skip offering this command line option if it isn't?